application/testing/CMakeLists.txt

```diff
@@ -252,6 +252,7 @@ f3d_test(NAME TestTextureMatCapWithTCoords DATA WaterBottle.glb ARGS --texture-m
 f3d_test(NAME TestConfigOrder DATA suzanne.ply ARGS CONFIG ${F3D_SOURCE_DIR}/testing/configs/config_order.json) # `.+` > `.*` alphabetically but overridden by the order
 f3d_test(NAME TestOutputStream DATA suzanne.ply ARGS --verbose=quiet --output=- REGEXP "^.PNG" NO_BASELINE NO_OUTPUT)
 f3d_test(NAME TestOutputStreamInfo DATA suzanne.ply ARGS --verbose=info --output=- REGEXP "redirected to stderr" NO_BASELINE NO_OUTPUT)
+f3d_test(NAME TestAliasCommand DATA dragon.vtu SCRIPT TestAliasCommand.txt --reference=${F3D_SOURCE_DIR}/testing/baseline/TestAliasCommand.png) # Alias command test
```

Comment:
still not the right name or location.

It should be put next to TestCommandScriptPrintOptions and named TestCommandScriptAlias